tempest-testing-docker/src/main/kotlin/app/cash/tempest/testing/DockerDynamoDbServer.kt

```diff
@@ -24,12 +24,14 @@ import com.github.dockerjava.api.model.Ports
 import com.google.common.util.concurrent.AbstractIdleService
 
 class DockerDynamoDbServer private constructor(
-  override val port: Int
+  override val port: Int,
+  private val releasePort: () -> Unit
```

Comment:
This is not a descriptive name; seems like you named it after its one implementor, instead of its function. I like 'onBeforeStartup` or `onPreStartup`.

Reply:
👍 I've renamed this to `onPreStartup`